proxy/hdrs/HTTP.cc

```diff
@@ -642,6 +653,19 @@ http_hdr_length_get(HTTPHdrImpl *hdr)
 /*-------------------------------------------------------------------------
   -------------------------------------------------------------------------*/
 
+int
+http_hdr_net_length_get(HTTPHdrImpl *hdr)
```

Comment:
Please add comments to document how this differs from the other similar header length functions.